capabilityreceiver/cmd/main.go

```diff
@@ -15,23 +15,46 @@ import (
 	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/endpointmanager/postgresql"
 )
 
-func main() {
-	err := config.SetupConfig()
+func setupCapStatReception(ctx context.Context, store *postgresql.Store) {
+	// Set up the queue for sending messages
+	qName := viper.GetString("capquery_qname")
+	messageQueue, channelID, err := accessqueue.ConnectToServerAndQueue(viper.GetString("quser"), viper.GetString("qpassword"), viper.GetString("qhost"), viper.GetString("qport"), qName)
 	helpers.FailOnError("", err)
+	log.Info("Successfully connected to Capability Statemsnts Queue!")
+	defer messageQueue.Close()
 
-	store, err := postgresql.NewStore(viper.GetString("dbhost"), viper.GetInt("dbport"), viper.GetString("dbuser"), viper.GetString("dbpassword"), viper.GetString("dbname"), viper.GetString("dbsslmode"))
+	err = capabilityhandler.ReceiveCapabilityStatements(ctx, store, messageQueue, channelID, qName)
 	helpers.FailOnError("", err)
-	log.Info("Successfully connected to DB!")
+}
 
+func setupVersionsReception(ctx context.Context, store *postgresql.Store) {
 	// Set up the queue for sending messages
-	qName := viper.GetString("capquery_qname")
+	qName := viper.GetString("versionsquery_response_qname")
 	messageQueue, channelID, err := accessqueue.ConnectToServerAndQueue(viper.GetString("quser"), viper.GetString("qpassword"), viper.GetString("qhost"), viper.GetString("qport"), qName)
 	helpers.FailOnError("", err)
-	log.Info("Successfully connected to Queue!")
+	log.Info("Successfully connected to Versions Response Queue!")
 	defer messageQueue.Close()
 
-	ctx := context.Background()
+	capQueryQueue, capQueryChannelID, err := accessqueue.ConnectToServerAndQueue(viper.GetString("quser"), viper.GetString("qpassword"), viper.GetString("qhost"), viper.GetString("qport"), "endpoints-to-capability")
```

Comment:
I think we should use viper.GetString("endpointinfo_capquery_qname") here for consistency